pkg/ui/workspaces/db-console/src/views/cluster/containers/clusterOverview/cluster.styl

```diff
@@ -237,7 +237,7 @@
       font-size 12px
       fill #7382a2
 
-    .axis .tick:nth-child(2n) text
+    .axis .tick:nth-child(2n+1) text
```

Comment:
this is the only thing that seems strange - did the numbering system for axises change?